apps/gdevelop/gdevelop.yml

```diff
@@ -0,0 +1,25 @@
+name: 'GDevelop'
+description: 'Create games without programming.'
+website: 'https://gdevelop-app.com/'
+repository: 'https://github.com/4ian/GDevelop'
+keywords:
+  - 'Game Editor'
+  - 'Open Source'
+  - 'Cross-Platform '
+  - 'Visual Programming'
+  - 'Online Editor'
+  - 'Piskel Sprite Editor'
+  - 'Google AdMob'
```

Comment:
Let's remove Google AdMob, it's not a defining feature of GDevelop, even if it's a nice extension.

Let's add:
  - 'gamedev'
  - 'game development'
  - 'game engine'
  - 'game making'
  - 'pixel-art'